packages/indexer/src/config/network.ts

```
@@ -113,7 +113,7 @@ export const getOpenseaNetworkName = () => {
     case 999:
       return "zora_testnet";
     default:
-      return null;
+      throw new Error(`Unsupported chainId ${config.chainId}`);
```

you changed the getOpenseaNetworkName to throw an error if its not supported. would that cause any issues where everever we use the method? (instead of failing silently)